### ISSUE TITLE
[ MD ] not detecting logouts after today's commits.

### ISSUE THREAD

**Issue by burstfreeze** [adiwajshing/Baileys#812]
The error handling/displaying is much better now however on device removal we don't get the `console.log('connection closed')` in example.ts anymore. May this be due to 428 overshadowing the 401?

**Comment by azudindaem** [comment]
I have no problem so far..

`Connection Update {
  connection: 'close',
  lastDisconnect: {
    error: Error: Connection Terminated by Server
        at WebSocket.<anonymous> (/home/webapps/md/node_modules/@adiwajshing/baileys-md/lib/Socket/socket.js:371:13)
        at WebSocket.emit (events.js:376:20)
        at /home/webapps/md/node_modules/@adiwajshing/baileys-md/lib/Socket/socket.js:229:35
        at Object.decodeFrame (/home/webapps/md/node_modules/@adiwajshing/baileys-md/lib/Utils/noise-handler.js:141:17)
        at WebSocket.onMessageRecieved (/home/webapps/md/node_modules/@adiwajshing/baileys-md/lib/Socket/socket.js:207:15)
        at WebSocket.emit (events.js:376:20)
        at Receiver.receiverOnMessage (/home/webapps/md/node_modules/ws/lib/websocket.js:983:20)
        at Receiver.emit (events.js:376:20)
        at Receiver.dataMessage (/home/webapps/md/node_modules/ws/lib/receiver.js:502:14)
        at Receiver.getData (/home/webapps/md/node_modules/ws/lib/receiver.js:435:17) {
      data: null,
      isBoom: true,
      isServer: false,
      output: [Object]
    },
    date: 2021-11-08T18:09:32.772Z
  }
} 2
Restarting app...
Express server on port 3007`

**Comment by canove** [comment]
I could not confirm this, clicking on "DISCONNECT" on phone fires the event normally here
```
[1636394502755] INFO: connection closed
    error: {
      "data": {
        "tag": "stream:error",
        "attrs": {
          "code": "401"
        },
        "content": [
          {
            "tag": "conflict",
            "attrs": {
              "type": "device_removed"
            },
            "content": null
          }
        ]
      },
      "isBoom": true,
      "isServer": false,
      "output": {
        "statusCode": 401,
        "payload": {
          "statusCode": 401,
          "error": "Unauthorized",
          "message": "Stream Errored"
        },
        "headers": {}
      }
    }
```

**Comment by burstfreeze** [comment]
The issue for me was the bad example.ts that's been addressed in #833
